Makefile

```diff
@@ -322,6 +322,11 @@ changelog-orbit:
 	sh -c "cat new-CHANGELOG.md orbit/CHANGELOG.md > tmp-CHANGELOG.md && rm new-CHANGELOG.md && mv tmp-CHANGELOG.md orbit/CHANGELOG.md"
 	sh -c "git rm orbit/changes/*"
 
+changelog-chrome:
+	sh -c "find ee/fleetd-chrome/changes -type file | grep -v .keep | xargs -I {} sh -c 'grep \"\S\" {}; echo' > new-CHANGELOG.md"
+	sh -c "cat new-CHANGELOG.md orbit/CHANGELOG.md > tmp-CHANGELOG.md && rm new-CHANGELOG.md && mv tmp-CHANGELOG.md orbit/CHANGELOG.md"
+	sh -c "git rm ee/fleetd-chrome/changes/*"
+
```

Comment:
@getvictor @lucasmrod is this the right place to add this code?
Do we not use a different Makefile for Orbit and Chrome?

Reply:
This is the correct place, we use one `Makefile` for everything.